### PR TITLE
PTL expect doesn't work if provided attributes with different case

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Bug/feature Description
* PTL's expect() is case sensitive and it doesn't work if verifying a list of attributes which have different case, e.g - a queue's 'Priority' attribute, if expect() is provided 'priority' instead, it fails to verify it, even though PBSPro is perfectly fine with a different case for attribute names.

#### Affected Platform(s)
* All Linux

#### Cause / Analysis / Design
* expect() was case sensitive when finding attributes in the list stat'd from PBS against the one provided by user

#### Solution Description
* Made expect() case insensitive when finding attributes in the list returned by PBS

#### Testing logs/output
* SmokeTest:
[smoketest.log](https://github.com/PBSPro/pbspro/files/2694379/smoketest.log)

* New testsuite TestExpect: 
[testexpect_before.log](https://github.com/PBSPro/pbspro/files/2694380/testexpect_before.log)
[testexpect.log](https://github.com/PBSPro/pbspro/files/2694381/testexpect.log)

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [X] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by agrawalravi90** [comment]
@arungrover and @bhroam can you please guys take a quick look at this before the break?

File: test/tests/selftest/pbs_expect.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        Test that when verifying attribute list containing attribute names
+        with different case, expect() is case insensitive
+        """
+        # Create a queue
+        a = {'queue_type': 'execution'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, 'expressq')
+
+        # Set the Priority attribute on the queue but provide 'p' lowercase
+        # Set other attributes normally
+        a = {'enabled': 'True', 'started': 'True', 'priority': 150}
```
Comment (bhroam) [inline test/tests/selftest/pbs_expect.py]: Don't you have to include queue_type?  If you don't, you can't set enabled and started.

Comment (anamikau) [reply test/tests/selftest/pbs_expect.py]: @bhroam he is already creating the queue with queue_type at line 53

Comment (bhroam) [reply test/tests/selftest/pbs_expect.py]: oops, missed that.  Thanks for pointing it out.

**Comment by agrawalravi90** [comment]
Thanks for the sign-off Bhroam.

Thanks for clarifying things Anamika.

@all one more review please, thanks

**Comment by agrawalravi90** [comment]
Thanks Anamika!

**Comment by agrawalravi90** [comment]
@anamikau and @bhroam this is ready for merge, thanks

**Comment by agrawalravi90** [comment]
@PBSPro/pbspro-maintainers this is ready for merge, please ignore Codacy, it's behaving weird these days. Thanks.

**Comment by anamikau** [comment]
Thanks @agrawalravi90 . Taking it for merge

**Pull request merged**